communitiesuk/funding-service-design-post-award-data-frontend
# BAU: Add VS Code Debugging

### Change description
Changes to allow VS Code debugger to attach to running docker container.

- Add debugpy to dev dependencies & update dependencies
- Add launch.json file for VS Code debugger
- Update Dockerfile with buildarg to allow build using requirements-dev.txt, defaults to use requirements.txt
- Rename requirements-dev to use hyphen for consistency across FSD repos
- Update github test.yml to match renamed requirements file




### How to test
Should run as before, only affects local development.

Instructions for using VS Code debugger on the code executing in the docker runner has been added to Docker Runner repo [README](https://github.com/communitiesuk/funding-service-design-post-award-docker-runner/blob/main/README.md) as part of [PR 23](https://github.com/communitiesuk/funding-service-design-post-award-docker-runner/pull/23
)
